data/moves.js

```diff
@@ -16674,11 +16674,15 @@ exports.BattleMovedex = {
 			this.add('-activate', source, 'move: Trick', '[of] ' + target);
 			if (myItem) {
 				target.setItem(myItem);
-				this.add('-item', target, myItem, '[from] Trick');
+				this.add('-item', target, myItem, '[from] move: Switcheroo');
+			} else {
+				this.add('-enditem', target, yourItem, '[silent]');
```

Comment:
Would that look like `'[silent] [from] move: Switcheroo'` or `'[silent]', '[from] move: Switcheroo'`?